### PR TITLE
Minor fixes to upgrade command

### PR DESCRIPTION
**Pull request by mbelur**
- Remove bootstrapPassword as a parameter to upgrade
- Add helm uninstall command
- Added note around backup and restore similar to the azure doc
- Add empty contents.lr for lektor to include rancher-prime in the build

### CODE REVIEW

File: content/rancher-prime/contents.lr
Comment (chajain) [inline content/rancher-prime/contents.lr:1]: Should we also specify not to install Rancher on target EKS cluster before the restore

Comment (mbelur) [reply content/rancher-prime/contents.lr:1]: > Should we also specify not to install Rancher on target EKS cluster before the restore

This line conveys that right? "Rancher Prime must be reinstalled on the target EKS cluster after restoring from the backup"
Do you want me to change it to " Rancher Prime must be reinstalled on the target EKS cluster after restoring from the backup but not before" ?

**Review by jeremy-moffitt — APPROVED** [review]
LGTM

File: content/rancher-prime/contents.lr
Comment (chajain) [reply content/rancher-prime/contents.lr:1]: ok with me

**Review by guangyee — APPROVED** [review]
looks good

**Review by chajain — APPROVED** [review]

**Pull request merged**